packages/jest-runner/corpus.ts

```diff
@@ -44,16 +55,56 @@ export class Corpus {
 
 	private createMissingDirectories() {
 		fs.mkdirSync(this._inputsDirectory, { recursive: true });
+		fs.mkdirSync(this._corpusDirectory, { recursive: true });
 	}
 }
 
-const buildInputsDirectory = (
+const buildDirectory = (
```

Comment:
I'd rename this to `directoryPathForTest`

Reply:
done